build/README.md

```diff
@@ -4,7 +4,7 @@ Docker Script for Building All-in-one-demo. Using this script you can build the
 
 ## Prerequisites
 
-- Operating system: **Ubuntu 18.04.**
+- Operating system: **Ubuntu* 20.04**
```

Comment:
Remove the * after Ubuntu, or escape it. Currently it is making the text italicized.

Reply:
Escaping asterisk.